.github/ISSUE_TEMPLATE/request-aws-iam-resources.md

```diff
@@ -0,0 +1,34 @@
+name: 'Request AWS IAM resources'
+description: 'Issue for requesting IAM accounts, groups, roles or policies'
```

Comment:
```suggestion
description: 'Issue for requesting IAM accounts'
```
I think we should have separate issues for groups, roles, and policies...just b/c the info we need will be different